packages/http/src/mocker/index.ts

```diff
@@ -20,7 +20,7 @@ import { map } from 'fp-ts/lib/Array';
 import { Do } from 'fp-ts-contrib/lib/Do';
 import { isNumber, isString, keyBy, mapValues, groupBy, get } from 'lodash';
 import { Logger } from 'pino';
-import * as typeIs from 'type-is';
+import { is } from 'type-is';
```

Comment:
```suggestion
import { is: typeIs } from 'type-is';
```

Just a suggestion, `is` may be too generic as it may be accidentally used like that `is(Vincenzo, listeningToGigiDagostino)`.

Reply:
I was thinking about making exactly the same suggestion but dropped it 😄

Reply:
Yes, it's a good idea